packages/brain/src/modules/cron/sendProofOfAttestation.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  DappnodeSigningProoverPostRequest,
+  Network,
+  Web3signerPostSignDappnodeRequest,
+  Web3signerPostSignDappnodeResponse,
+} from "@stakingbrain/common";
+import { Web3SignerApi, DappnodeSigningProover } from "../apiClients/index.js";
+import { BrainDataBase } from "../db/index.js";
+import logger from "../logger/index.js";
+
+/**
+ * Send the proof of attestation to the dappnode-signatures.io domain
+ */
+export async function sendProofOfAttestation(
+  signerApi: Web3SignerApi,
+  brainDb: BrainDataBase,
+  dappnodeSigningProoverApi: DappnodeSigningProover,
+  network: Network
+): Promise<void> {
+  try {
+    // Get the proofs of attestation from the signer
+    const proofsOfAttestations = await getProofsOfAttestations(
+      signerApi,
+      brainDb,
+      network
+    );
+    if(proofsOfAttestations.length === 0) return
+    await dappnodeSigningProoverApi.sendProofOfAttestation(
+      proofsOfAttestations
+    );
+  } catch (e) {
+    logger.error(`Error sending proof of attestation: ${e.message}`);
+  }
+}
+
+/**
+ * Get the proofs of attestation from the signer
+ * for all the pubkeys in the db
+ */
+async function getProofsOfAttestations(
+  signerApi: Web3SignerApi,
+  brainDb: BrainDataBase,
+  network: Network
+): Promise<DappnodeSigningProoverPostRequest[]> {
+  const signerDappnodeSignRequest: Web3signerPostSignDappnodeRequest = {
+    type: "PROOF_OF_VALIDATION",
+    platform: "dappnode",
+    timestamp: new Date().toISOString(),
+  };
+  // get pubkeys detauls from db
+  const dbPubkeysDetails = brainDb.getData();
+  // For each pubkey, get the proof of attestation from the signer
+  const proofsOfAttestations = await Promise.all(
+    Object.keys(dbPubkeysDetails).map(async (pubkey) => {
```

Comment:
maybe it is a good idea to wait a bit between each request to the signer to ensure we dont overload it. just a thought

Reply:
already tested in the web3signer with hundred of keys, seems to be handled without issues